packages/terra-button/src/Button.jsx

```diff
@@ -326,6 +350,7 @@ class Button extends React.Component {
         onFocus={this.handleFocus}
         href={href}
         ref={refCallback}
+        aria-pressed={isSelectable ? this.state.isSelected : undefined}
```

Comment:
what is the SR reading out when aria-pressed is set to True ?

Reply:
It says "Toggle button pressed, <button_label>"